src/go/go.mod

```diff
@@ -1,4 +1,4 @@
-module apiview
+module apiviewgo
```

Comment:
Why does the module (and thus the .exe) name need to change?  I'm not pushing back, just want to understand why.

Reply:
You just answered my question.

Reply:
APIView name was already in use within APIView publish package but so far this binary was not packed by dotnet and go parser was not causing any name collision. But latest dotnet version has started building and including this console test within core APIView project. We cannot change this core APIView since that's the base of APIView tool itself. I also feel it's more clear as other language parser when we  have language name as part of parser to avoid collision.